File: C#/practice1/challenge8.cs
using System;

namespace C_.practice1
{
    public class challenge8
    {
        public challenge8()
        {

        }

        public void Challenge_8()
        {
            Console.WriteLine("Challenge 8");
            Console.WriteLine("Waiter, the bill please.");
            string bill = Console.ReadLine();
            double Bill = double.Parse(bill);
            Console.WriteLine("How many are we are?");
            string persons = Console.ReadLine();
            int Persons = int.Parse(persons);
            Console.WriteLine("Guys, what's the percent for the tip?");
            string tipPercent = Console.ReadLine();
            double TipPercent = double.Parse(tipPercent);

            double taxesPercent = 0.16;
            Console.WriteLine("Taxes % {0}", taxesPercent);

            var totalTip = Bill * TipPercent;
            Console.WriteLine("Tip $ {0}", totalTip);
            var totalTaxes = Bill * taxesPercent;
            Console.WriteLine("Taxes $ {0}", totalTaxes);

            var TotalBill = Bill + totalTip + totalTaxes;
            Console.WriteLine("Account total, included tip and taxes: $ {0}", TotalBill);
            var TotalByPerson = TotalBill / Persons;
            Console.WriteLine("Account total by person, included tip and taxes: $ {0}", TotalByPerson);



        }
    }
}

File: Javascript/challenge8.js
console.log('Challenge 8');
let bill = prompt('Waiter, the bill please.');
console.log('Bill:' + parseFloat(bill));
let persons = prompt('How many are we are?');
console.log(persons + ' Persons.');
let tipPercent = prompt("Guys, what's the percent for the tip?");
console.log('Tip % ' + tipPercent);

const taxesPercent = 0.16;

const Total = (b, t, taxes) => {
  let Tip = parseFloat(b) * parseFloat(t);
  console.log('Tip $' + Tip);

  let TotalTaxes = parseFloat(b) * parseFloat(taxes);
  console.log('Taxes % ' + taxesPercent);
  console.log('Taxes $ ' + TotalTaxes);

  let Total = parseFloat(b) + parseFloat(Tip) + parseFloat(TotalTaxes);
  return Total;
};

const total = Total(bill, tipPercent, taxesPercent);
console.log('Account Total, included tip and taxes: $' + total.toFixed(2));

const dividedByPerson = (persons, total) => {
  let TotalByPerson = total / persons;
  return TotalByPerson;
};

const totalByPerson = dividedByPerson(persons, total);
console.log('Account total by person, included tip and taxes: $' + totalByPerson.toFixed(2));
